java/src/main/native/src/TableJni.cpp

```diff
@@ -46,6 +46,7 @@
 #include <cudf/utilities/span.hpp>
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/mr/device/device_memory_resource.hpp>
+#include <rmm/resource_ref.hpp>
```

Comment:
Is this needed, or were other changes meant to be made in this file?

Reply:
It was added by my script but then I undid the changes made by the script, so I need to remove this.

Reply:
Removed.